test/test_torch.py

```diff
@@ -17229,7 +17233,7 @@ def genf_float(x, y):
             _test_mm(n, m, p, dtype, genf)
 
     @onlyCPU
-    @dtypes(torch.float)
+    @dtypes(*(torch.testing.get_all_complex_dtypes() + [torch.float, torch.double]))
```

Comment:
NIT: Could we unify

```
torch.float32, torch.float64, torch.cfloat, torch.cdouble
```

vs

```
torch.testing.get_all_complex_dtypes() + [torch.float, torch.double]
```

Reply:
do you mean, replace all instances of `torch.float32, torch.float64, torch.cfloat, torch.cdouble` with `torch.testing.get_all_complex_dtypes() + [torch.float, torch.double]` ?

Reply:
I mean the opposite:
```
torch.float32, torch.float64, torch.cfloat, torch.cdouble
```
looks better than
```
*(torch.testing.get_all_complex_dtypes() + [torch.float, torch.double])
```

Reply:
I think it's better to use `torch.testing.get_all_complex_dtypes()` since in case we add more complex dtypes in future (for example, support for torch.chalf or others), we wouldn't have to update the tests one by one to add tests for that complex dtype. Just including it in `torch.testing.get_all_complex_dtypes()` would suffice.